benchmarks/src/main/java/io/grpc/benchmarks/driver/LoadClient.java

```diff
@@ -198,7 +206,8 @@ void start() {
             }
           } else if (config.getClientType() == Control.ClientType.ASYNC_CLIENT) {
             if (config.getRpcType() == Control.RpcType.UNARY) {
-              r = new AsyncUnaryWorker(asyncStubs[i % asyncStubs.length]);
+              r = new GapicAsyncUnaryWorker(gapicClients[i % gapicClients.length]);
+              // r = new AsyncUnaryWorker(asyncStubs[i % asyncStubs.length]);
```

Comment:
We have this before but it is also in a PR (not merged) https://github.com/shinfan/grpc-java/pull/1